server/store/Scanning/cloudOCR.js

```diff
@@ -64,26 +76,45 @@ const transcriptParser = async paragraph => {
       break;
 
     // GET SEMESTER
-    case words.includes('SEMESTER') && words.length === 3: {
-      // 0: Taken
-      // 1: In Progress
-      const semester = {
-        term: words[0],
-        year: parseInt(words[2], 10),
-        courses: [],
-        status: 0,
-      };
-      if (!startingTermFound) {
-        startingSem = semester;
+    case semesterSeason.some(semester => words[0].includes(semester)) &&
+      words.includes('SEMESTER'): {
+      if (!words[2]) {
```

Comment:
If there is no year found in the sentence. Mark it `missingStartYear = true` to get the year in the next sentence.